apps/orams/components/SellerCatalogue/SellerCatalogue.js

```diff
@@ -1,85 +1,42 @@
 /* eslint-disable */
 import React, { Component } from 'react'
-import { withRouter } from 'react-router-dom'
 import { connect } from 'react-redux'
 import RegionsAccordionRadioList from 'orams/components/AccordionRadioList/RegionsAccordionRadioList'
 import CategoriesAccordionRadioList from 'orams/components/AccordionRadioList/CategoriesAccordionRadioList'
 import ResultsTable from 'orams/components/ResultsTable/ResultsTable'
-
+import LoadingIndicatorFullPage from 'shared/LoadingIndicatorFullPage/LoadingIndicatorFullPage'
+import { loadRegions, loadServices } from 'orams/actions/sellerCatalogueActions'
+import PageAlert from '@gov.au/page-alerts'
 import styles from './SellerCatalogue.scss'
 
-const regionsData = {
-  regions: [
-    {
-      mainRegion: 'NSW',
-      subRegions: [{ name: 'Metro' }, { name: 'Mid North' }]
-    },
-    {
-      mainRegion: 'ACT',
-      subRegions: [{ name: 'Metro' }, { name: 'West' }]
-    }
-  ]
-}
+class SellerCatalogue extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {}
+  }
+
+  componentDidMount() {
+    this.props.loadRegionsData()
+    this.props.loadServicesData()
+  }
 
-const categoriesData = {
-  categories: [
-    {
-      mainCategory: 'Rehabilitation services',
-      subCategories: [{ name: 'Ergonomic review' }, { name: 'Early intervention rehabilitation program' }]
-    },
-    {
-      mainCategory: 'Medical services',
-      subCategories: [{ name: 'Example service' }, { name: 'Second example service' }]
+  tableSection() {
+    if (this.props.currentlySending) {
+      return <LoadingIndicatorFullPage />
     }
-  ]
-}
 
-const tableData = {
-  alert: {
-    type: 'info',
-    message: 'This service is a fixed fee - inclusive of travel, assessment and report'
-  },
-  suppliers: [
-    {
-      name: 'APM',
-      price: '$330.00',
-      phone: '08 9486 1244',
-      email: 'ORAMS.DHS@apm.net.au'
-    },
-    {
-      name: 'Carfi',
-      price: '$192.50',
-      phone: '03 8648 0900',
-      email: 'atoreferral@ipar.com.au'
-    },
-    {
-      name: 'Injury Treatment',
-      price: '$192.50',
-      phone: '1300 622 734',
-      email: 'atoreferral@injurytreatment.com.au'
-    },
-    {
-      name: 'IPAR Rehabilitation',
-      price: '$192.28',
-      phone: '1300 762 989',
-      email: 'referral@rehabmanagement.com.au'
-    },
-    {
-      name: 'Rehab Management',
-      price: '$192.50',
-      phone: '1300 762 989',
-      email: 'referrals@rehabmanagement.com.au'
+    if (this.props.errorMessage) {
```

Comment:
I could not get this to display - are you checking the right prop?

Reply:
Yes it is, same prop as in marketplace. Can we merge this?